TUM Campus App/CardCells/CafeteriaCollectionViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CafeteriaCollectionViewCell.swift
+//  Campus
+//
+//  Created by Tim Gymnich on 03.05.18.
+//  Copyright © 2018 LS1 TUM. All rights reserved.
```

Comment:
Do we need this copy right? What is LS1? Why is this code copyrighted to Prof. Brügge?

Reply:
Xcode does this... Should I just replace this with TCA ???

Reply:
No, please just use the GPLv3: https://github.com/TCA-Team/iOS/blob/master/LICENSE

Short part is this for example: https://gist.github.com/ilg-ul/aa6de96c993719667f07